.ci/install_firecracker.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+arch=$("${cidir}"/kata-arch.sh -d)
+source "${cidir}/lib.sh"
+
+if [ "$arch" != "x86_64" ]; then
+	die "Static binaries for Firecracker only available with x86_64."
+fi
+
+if [ "$KATA_DEV_MODE" = true ]; then
+	die "Dangerous to set KATA_DEV_MODE"
```

Comment:
Nit: This is rather misleading I think as developers *should* set that to avoid potentially dangerous operations from running on their systems. How about:

```sh
die "KATA_DEV_MODE set so not running test"
```

However, @egernst and @mcastelino may have further thoughts on how this script is expected to work.